tests/integration/test_logic_pipeline.py

```
@@ -0,0 +1,232 @@
+        🌲 Test Integration: Logic Results -> Forest Plot
+        Verifies that logistic regression output can be visualized as a Forest Plot
+        """
+        df, y = sample_medical_data
+        
+        # 1. Run Logistic Regression via analyze_outcome (which returns structured results)
+        # analyze_outcome combines run_binary_logit and formatting
+        full_df = df.copy()
+        full_df['disease'] = y
+        
+        # Use analyze_outcome to get Odds Ratios (OR) and Confidence Intervals
+        html_table, or_results, aor_results, int_results = analyze_outcome(
```

Ruff (0.14.10)

Unpacked variable `html_table` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)